cypress/tests/e2e/e2e.cy.ts

```diff
@@ -0,0 +1,97 @@
+import { signInOkta } from '../../lib/signInOkta';
+
+describe('E2E with Okta', () => {
+	beforeEach(() => {
+		cy.clearAllCookies();
+		signInOkta();
+		cy.get('h1', {
+			timeout: 30000,
```

Comment:
Seems like a very long timeout.  And could this value be extracted or do you foresee this varying independently in different places?

Reply:
This turns out to be quite a safe value for the headless Cypress browser running in GitHub Actions, communicating through an Nginx proxy with the CODE APIs 🙃. It's _slow_. I've extracted it out to a Cypress.config call though, good idea.